_posts/2019-02-07-apis-are-watching-you.markdown

```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: "APIs and UIs"
+title: "APIs Are Watching You"
```

Comment:
How about something less scary? 😄 
I'm trying to think of something that ryhmes like "APIs for Your Eyes" or "Modernize with APIs". Maybe you can come up with something less cheesy.

Reply:
Not a hall and oats fan? 😄